storage/onpremise/mongodb/variables.tf

```diff
@@ -51,10 +59,50 @@ variable "persistent_volume" {
     wait_until_bound = optional(bool, true)
   })
   default = null
+
+  # For retrocompatibility concerns (AWS deployment), this variable must be null for now
+  validation {
+    condition     = var.persistent_volume == null
+    error_message = <<EOT
+    "For now, 'persistent_volume' variable must be null.
+    Custom persistent volume definition for MongoDB is soon to be implemented"
+    EOT
+  }
+}
+
+# Not used yet (there for retrocompatibility reasons)
+variable "security_context" {
+  description = "Security context for MongoDB pods"
+  type = object({
+    run_as_user = number
+    fs_group    = number
+  })
+  default = {
+    run_as_user = 999
+    fs_group    = 999
+  }
+}
+
+variable "mtls" {
+  description = "Whether to deploy MongoDB with mTLS"
+  type        = bool
+  default     = false
+  validation {
+    condition     = !var.mtls
+    error_message = "For now, 'mTLSEnabled' must be false as mTLS is not supported yet"
+  }
+}
```

Comment:
I am not sure this should be exposed, as it was not configurable before. You can add it back later on, once it is implemented.